src/lib/components/support-card/support-card.svelte

```diff
@@ -88,35 +134,28 @@
       <div class="circle">
         <Heart style="height: 3rem; width: 3rem; fill: var(--color-primary)" />
       </div>
-      <div>
-        <ProjectAvatar {project} size="large" outline />
-      </div>
+      {#if project}
+        <div>
+          <ProjectAvatar {project} size="large" outline />
+        </div>
+      {/if}
     </div>
   </div>
   <h2 class="pixelated">Become a supporter</h2>
   <p>
-    {#if isSupportingProject}
-      You're already supporting <span class="typo-text-bold">{project.source.repoName}</span> with your
-      Drip List.
+    {#if isSupporting}
+      You're already supporting this with your Drip List.
     {:else}
-      Add <span class="typo-text-bold">{project.source.repoName}</span> to your Drip List to support
-      this project.
+      Add this {project ? 'project' : ''} to your Drip List to flexibly support it with an ongoing contribution.
     {/if}
   </p>
   <Button
     on:click={handleSupportButton}
-    disabled={isSupportingProject}
+    disabled={isSupporting || isOwner}
     size="large"
     icon={Plus}
-    variant="primary">{dripList === null ? 'Create your Drip List' : 'Add to Drip List'}</Button
+    variant="primary">{ownDripList === null ? 'Create your Drip List' : 'Add to Drip List'}</Button
```

Comment:
maybe nice if they're already supporting the button is disabled with checkmark icon and "Added to your Drip List"?

Reply:
yeah tru

Reply:
though irrelevant very soon when we launch the multi drip list feature, so i'd keep it like this for now!